setup.py

```diff
@@ -36,16 +36,17 @@
       install_requires=['numpy>=1.9.1',
                         'scipy>=0.14',
                         'six>=1.9.0',
-                        'h5py>=2.7.1',
                         'pyyaml',
-                        'keras_applications==1.0.4',
-                        'keras_preprocessing==1.0.1'],
+                        'h5py',
```

Comment:
is this compatible with older versions of h5py? we should probably keep the version requirement of h5py to prevent any breaking builds

Reply:
this change is from keras, let's keep it and it's not causing any build breaks: https://github.com/keras-team/keras/blob/master/setup.py